spec/README.md

```diff
@@ -288,6 +288,15 @@ full list of relevant test files.
 
 ## Client-Side Encryption
 
+NOTE: Client-side encryption tests require an enterprise build of MongoDB
+server version 4.2 or higher. These builds of the MongoDB server come packaged with
+mongocryptd, a daemon that is spawned by the driver during automatic encryption.
+The client-side encryption tests require the mongocryptd binary to be in the
+system path.
```

Comment:
@p-mongo Mention that the texts expect the mongocryptd binary to be in the system path